src/display/editor/ink.js

```diff
@@ -677,11 +734,22 @@ class InkEditor extends AnnotationEditor {
    * onpointerdown callback for the canvas we're drawing on.
    * @param {PointerEvent} event
    */
-  canvasPointerdown(event) {
+  canvasPointerDown(event) {
```

Comment:
Here, and elsewhere, please avoid increasing the size of the diff (and changing blame) by not unnecessarily renaming methods.